dev/grid.html

```diff
@@ -307,7 +323,14 @@ <h4>Details for ${model.item.name}</h4>
       });
 
       // Theme Controls
-      const themeControls = ['no-border', 'no-row-borders', 'column-borders', 'row-stripes', 'wrap-cell-content'];
+      const themeControls = [
```

Comment:
Now when https://github.com/vaadin/web-components/pull/10198 is merged, this change needs to be applied to `dev/playground/grid.html` instead.